src/google/protobuf/generated_message_reflection.cc

```diff
@@ -1223,6 +1223,38 @@ void Reflection::SetString(Message* message, const FieldDescriptor* field,
 }
 
 
+void Reflection::SetString(Message* message, const FieldDescriptor* field,
+                           const std::string&& value) const {
```

Comment:
@reed-lau rvalue references cannot be const as they cannot be moved. The overloaded functions rvalue parameter should be SetString(..,..,std::string&& value) without the const!